### DISCUSSION TITLE
What is the reccommended way to convert the Freemind v1.0.1 format .mms to Freeplane .mms?

### DISCUSSION THREAD

**Discussion by Coderific** [freeplane/freeplane#533] [Help]
There is an error that comes up when I open the Freemind .mms that says to go to the Freemind site on SourceForge, but that link doesn't exist.

I have to convert 100+ maps to Freeplane.

I tried copying the edges and nodes from Freemind, and pasted them into Freeplane, but they didn't come across ok.

I opened Freemind .mms in Freeplane, but get the error above.

What is the recommend way to do this?

**Comment by dpolivaev** [comment]
Try to handle the error as a warning which can be ignored. If the mind map is opened by Freeplane and you do not see any information loss you can just save it and use it. But make sure to create backups of the original files.

**Reply by Coderific** [reply]
What is an easy way to determine if there was data loss?

Is there some kind of utility that counts nodes or things? Some of my mind maps have more than 10,000 nodes.

**Reply by dpolivaev** [reply]
I would expect no problems if you do not use FreeMind clones. But in doubt, it is better to have a backup just in case something goes wrong.

**Reply by EdoFro** [reply]
> What is an easy way to determine if there was data loss?
> 
> Is there some kind of utility that counts nodes or things? Some of my mind maps have more than 10,000 nodes.

use command in menu **File->Map statistics…**

**Reply by dpolivaev** [reply]
Usually, if the map was loaded without error messages it must have been loaded completely.
The first message you get is not an error message but just a warning. It does not mean any data loss by itself. You can choose to ignore it.

**Comment by Coderific** [comment]
Thank you for your response and the good information.
